invenio_app_rdm/config.py

```diff
@@ -939,6 +939,8 @@ def github_link_render(record):
 RDM_COMMUNITIES_ROUTES = {
     "community-detail": "/communities/<pid_value>/records",
     "community-home": "/communities/<pid_value>/",
+    "community-static-page": "/communities/<pid_value>/info",
```

Comment:
cc @slint - I think this should be ``/communities/<pid_value/pages/<path:..>`` so we can add any number of pages without code deployment.

Reply:
Agree, so that we can "generically" add arbitrary pages under this endpoint using `invenio-pages`